vhdl_syntax/src/syntax/node_kind.rs

```diff
@@ -4,6 +4,12 @@
 //
 // Copyright (c)  2025, Lukas Scheller lukasscheller@icloud.com
 
+#[derive(PartialEq, Eq, Copy, Clone, Debug)]
+pub enum InternalNodeKind {
+    ActualPartTokens,
+    SubtypeIndicationOrExpressionTokens,
```

Comment:
I would probably just make this an abstract `Tokens` (or `RawTokens` or so). I think that the context can unambiguously be derived from where the tokens are stored.